### ISSUE TITLE
The loaded pre-trained TF-slim model (inceptionV3) cannot achieve the same results as it states

### ISSUE THREAD

**Issue by Openning07** [ppwwyyxx/tensorpack#421]
Recently, I'm working on my research based on the inceptionV3 model. But somehow, I cannot achieve the same results as it claims on ILSVRC12 ...
DETAILS: I've managed to load all necessary variables from a checkpoint file of a pretrained inceptionV3 model by applying TF-Slim defined modules and naming the tensor variables the same in the checkpoint file (because the log after loading is: **The following variables are in the checkpoint, but not found in the graph: global_step:0**). Besides that, I changed the data pipeline (both the train pipeline and the test one) to make it the same as the preprocessing of that pretrained inceptionV3 model (available on https://github.com/tensorflow/models/blob/master/slim/preprocessing/inception_preprocessing.py). I've checked it several times, but still fail to achieve the same performance: the stated results are 78.0 (top1) | 93.9 (top5), but I can just get 70.0 (top1) | 90.8 (top5).

Have you ever met the same or similar problem?

Besides, have you ever met some problems in training a neural network defined with TF-Slim module? Since there are a few conflict between API of TF-Slim and that of Tensorpack. For example, batch_norm in TF-Slim doesn't use scale (i.e., gamma) by default, while in Tensorpack, batch_norm would use scale by default ...

**Comment by ppwwyyxx** [comment]
This is not something I can help with because any tiny difference between your code and the original ones can cause this and I couldn't know what that difference is.
You can actually just call `slim.inception_v3` inside tensorpack, and this might help reduce the differences.

**Comment by Openning07** [comment]
Thank you for such a quick response!
I would check even further to try to solve it...
Thanks anyway!

**Comment by ppwwyyxx** [comment]
btw, our examples contain an inceptionv3 code which reaches 74% accuracy. This is what you should __expect__ to get. 78% will need super large batch size (e.g. 1600). See Google's comments (https://github.com/tensorflow/models/blob/master/research/inception/inception/inception_train.py#L66)
